### ISSUE TITLE
Storing byte[] as BLOB

### ISSUE THREAD

**Issue by hamen** [pushtorefresh/storio#496]
Hi all
I'm trying to figure out how to store a `byte[]` as a BLOB.
Right now I get:

```
Unsupported type of field for StorIOSQLiteColumn annotation,
if you need to serialize/deserialize field of that type -> 
please write your own resolver: Unsupported type: byte[]
```

Any tip? Thanks.

**Comment by artem-zinnatullin** [comment]
Damn, just forgot about blobs (didn't use blobs in sqlite for several years). I'll add it to annotation processor today-tomorrow or if you want to help us a little — you can open a PR, annotation processor is no magic :)

At the moment you can use custom operation resolver.

**Comment by hamen** [comment]
Yeah.... BLOB... I know... weird requirements.
Anyhow, thank you. I'll look into it.

**Comment by artem-zinnatullin** [comment]
@hamen JFYI I am working on PR, so you can just sit and relax :)

**Comment by artem-zinnatullin** [comment]
@hamen just released 1.3.0 with support for `byte[]` via Annotation Processor, should be available in 10-20 minutes!

**Comment by hamen** [comment]
Awesome! First thing in the morning! :smile:
